pandas/core/reshape/merge.py

```diff
@@ -541,23 +545,23 @@ class _MergeOperation:
 
     def __init__(
         self,
-        left,
-        right,
-        how="inner",
+        left: FrameOrSeries,
+        right: FrameOrSeries,
```

Comment:
The docstring implies these are DataFrames

Reply:
the docstring is incorrect, ill update it